resources/options.js

```diff
@@ -76,50 +66,75 @@ function restoreOptionsAvailability() {
 
 // Shows the sub-options.
 function showSubOptions() {
-    $('#security_sub').show();
-    $('#privacy_sub').show();
-    $('#hints_sub').show();
+    for(var i = 0; i < subOptions.length; i++) {
+        subOptionsElement(i).show();
+    }
 }
 
 // Hides the sub-options.
 function hideSubOptions() {
-    $('#security_sub').hide();
-    $('#privacy_sub').hide();
-    $('#hints_sub').hide();
+    for(var i = 0; i < subOptions.length; i++) {
+        subOptionsElement(i).hide();
+    }
 }
 
 // Shows the option cards.
 function showCards() {
-    $('#security').show();
-    $('#privacy').show();
-    $('#hints').show();
+    for(var i = 0; i < cards.length; i++) {
+        cardElement(i).show();
+    }
 }
 
 // Hides the option cards.
 function hideCards() {
-    $('#security').hide();
-    $('#privacy').hide();
-    $('#hints').hide();
+    for(var i = 0; i < cards.length; i++) {
+        cardElement(i).hide();
+    }
 }
 
 // Switches availability of options.
 function switchDisable() {
-    $('#tabs').disabled = !$('#tabs').disabled;
-    $('#comments').disabled = !$('#comments').disabled;
-    $('#peerComments').disabled = !$('#peerComments').disabled;
-    $('#focus').disabled = !$('#focus').disabled;
-    $('#username').disabled = !$('#username').disabled;
-    $('#repo').disabled = !$('#repo').disabled;
-    $('#file').disabled = !$('#file').disabled;
+    for(var i = 1; i < options.length; i++) {
```

Comment:
"switch" flips the values of the disabled property for each option, excluding the main option.
"restore" changes the values of the disabled property for each option, based on the value of main option.
So how do you want to use only one for-loop? (As what's happening inside the for-loops is a bit different.)